zerver/lib/message.py

```diff
@@ -87,6 +87,16 @@ class UnreadMessagesResult(TypedDict):
     old_unreads_missing: bool
 
 
+@dataclass
+class RenderedMessage:
```

Comment:
I think this needs to subclass `Message` if you're going to use it as written.

Reply:
From a [chat.zulip.org message](https://chat.zulip.org/#narrow/stream/49-development-help/topic/Message.20not.20as.20Any), it sounds like this intentionally avoids subclassing `Message` to avoid being recognized as a Django model.